src/ServerScriptService/InstanceComponents/Misc/Character.lua

```diff
@@ -13,6 +13,8 @@ function Character.new(instance: Instance)
 
 	-- Add CameraLook tag to the character
 	CollectionService:AddTag(instance, "CameraLook")
+	instance:SetAttribute("AutoRotate", true)
+	instance:SetAttribute("FaceCursor", true)
```

Comment:
Load dependent attributes first (It's idiomatic - they will replicate out of order anyways)

Attributes, then tag.